tools/mason/MasonBuild.chpl

```diff
@@ -52,6 +61,10 @@ proc masonBuild(args) {
       else if arg == '--example' {
         example = true;
       }
+      // passed to UpdateLock
+      else if arg == '--no-update' {
```

Comment:
Long term, I could imagine an environment variable enabling this option being useful. Nothing to change here, but I've added this desire to https://github.com/chapel-lang/chapel/issues/7106